<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDockWidget</class>
 <widget class="QDockWidget" name="MainDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>244</width>
    <height>114</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QQ</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLabel" name="label_MyAvatar">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_MyStateShow">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>40</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>16777215</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_MyNick">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>0</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>昵称------------------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MySignature">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>35</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>个性签名||||||||||||5345364--------------------------------2q3254356--------|||||||||||||||||||||</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>90</y>
      <width>231</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_BuddyList">
     <attribute name="title">
      <string>联系人</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget_BuddyList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>291</width>
        <height>321</height>
       </rect>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>我的好友</string>
       </property>
       <item>
        <property name="text">
         <string>小红</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>小明</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>小王</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>朋友</string>
       </property>
       <item>
        <property name="text">
         <string>小虎</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>小赵</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>校长</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>同学</string>
       </property>
       <item>
        <property name="text">
         <string>张同学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>里同学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>州同学</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>家人</string>
       </property>
       <item>
        <property name="text">
         <string>妹妹</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>各个</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>姐姐</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>弟弟</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>陌生人</string>
       </property>
       <item>
        <property name="text">
         <string>不认识</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>黑名单</string>
       </property>
       <item>
        <property name="text">
         <string>王八蛋</string>
        </property>
       </item>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_GroupList">
     <attribute name="title">
      <string>群/讨论组</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_GroupList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>351</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>同学群</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>老乡群</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>技术群</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>朋友群</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRecentList">
     <attribute name="title">
      <string>最近联系人</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_RecentList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>231</width>
        <height>391</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>各个</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>妹妹</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>小红</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>小李</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
